addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -110,11 +110,12 @@ def _get_partner_party_vals(self, partner, role):
         vals['endpoint_id'] = partner.vat
         vals['endpoint_id_attrs'] = {'schemeID': COUNTRY_EAS.get(partner.country_id.code)}
 
-        if partner.country_code == 'NO' and 'l10n_no_bronnoysund_number' in partner._fields:
-            vals.update({
-                'endpoint_id': partner.l10n_no_bronnoysund_number,
-                'endpoint_id_attrs': {'schemeID': '0192'},
-            })
+        if partner.country_code == 'NO':
+            vals['endpoint_id_attrs'] = {'schemeID': '0192'}
```

Comment:
I'm sorry for my stupid question but why do we need to do that as the value is already correct:
https://github.com/odoo/odoo/blob/04593b265b765f7a6ee079f21bad6a1cf0e5d094/addons/account_edi_ubl_cii/models/account_edi_common.py#L84

Reply:
You're right, I missed that. But starting from 16.2, the field is read from the `partner.peppol_endpoint` which could be anything. Indeed, the `EAS_MAPPING` is used to try to fill this field (as well as `partner.peppol_eas`) but it can then be modified by the user.

Reply:
So can we remove it in 15.0? :thinking:

Reply:
Yes, please @JordiMForgeFlow :)

Reply:
@JulienVR done :)